Client/Cargo.toml

```diff
@@ -5,9 +5,6 @@ edition = "2021"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
-[workspace]
-resolver = "2"
```

Comment:
What's the logic behind the removal of this? Does it relate to the 3k lost lines in the Cargo.lock?